.travis.yml

```diff
@@ -25,12 +25,15 @@ matrix:
       env: SYMFONY_VERSION=2.8.*@dev
     - php: 5.6
       env: SYMFONY_VERSION=3.0.*@dev
+    - php: 5.6
+      env: TWIG_VERSION=2.0@dev
     - php: 5.3
```

Comment:
The requested package twig/twig could not be found in any version, there may be a typo in the package name.